### PR TITLE
notebooks: remove elm compute component

### PR DESCRIPTION
**Pull request by limitedmage**
This removes all code related with the Notebook compute component, including the backend API, Elm build, etc.

## Test plan

- [x] An existing notebook with a compute block should still load (without rendering that block)
- [x] Editing that existing notebook should save the notebook without the compute block

### CODE REVIEW

File: enterprise/internal/notebooks/validate.go [hunk 1]
```diff
@@ -6,8 +6,7 @@ func validateNotebookBlock(block NotebookBlock) error {
 	if block.Type != NotebookQueryBlockType &&
 		block.Type != NotebookMarkdownBlockType &&
 		block.Type != NotebookFileBlockType &&
-		block.Type != NotebookSymbolBlockType &&
```
Comment (eseliger) [inline enterprise/internal/notebooks/validate.go:9]: what do we do about existing notebooks with these blocks? Would they fail to save?

**Review by eseliger — APPROVED** [review]
looks like some previously ignored files snuck into the diff!
Screenshot 2022-12-07 at 19 31 00@2x

File: enterprise/internal/notebooks/validate.go (hunk 1 above)
Comment (limitedmage) [reply enterprise/internal/notebooks/validate.go:9]: I've changed `backend.ts` to [delete any invalid blocks before saving](https://github.com/sourcegraph/sourcegraph/pull/45360/files#diff-c52e6d73dfeb161e746aef13c7138a5697ae061e2abb0dd7943d3a29f18880fd).

**Comment by limitedmage** [comment]
> looks like some previously ignored files snuck into the diff!

Good catch, thanks! This is because I removed the `.gitignore` rule but didn't delete the previously-ignored files 🤦‍♀️

File: enterprise/internal/notebooks/validate.go (hunk 1 above)
Comment (limitedmage) [reply enterprise/internal/notebooks/validate.go:9]: I debated where to do this (backend? frontend? on fetch? on save?) and decided this was the easiest way.

Comment (eseliger) [reply enterprise/internal/notebooks/validate.go:9]: works for me as long as nothing breaks - doesn't sound like it :)

Comment (limitedmage) [reply enterprise/internal/notebooks/validate.go:9]: It's working for me at least from what I've tested (test cases in the description above)

**Review by rvantonder — APPROVED** [review]
thanks! you got to this before I did 😆

**Comment by valerybugakov** [comment]
Nice, thank you @limitedmage!

**Pull request merged**